client/app/components/game-client.js

```diff
@@ -0,0 +1,64 @@
+import Ember from 'ember';
+import GameChannel from 'vilive/utils/game-channel';
+
+export default Ember.Component.extend({
+  init() {
+    this.set('messages', Ember.makeArray());
+    GameChannel.init(this.get('session-manager').currentToken());
+    GameChannel.on('event', payload => {
+      this.putMessage(true, payload.body);
```

Comment:
I'm really not a fan of boolean arguments. From this usage I have no idea what it does. I would rather see `putMessage()` take an object that looks more like:

```javascript
{
    origin: "player",
    text: "join game"
}
```